R/rdeephaven/prototyping/tmp.R

```diff
@@ -0,0 +1,31 @@
+Rcpp::compileAttributes()
+roxygen2::roxygenise()
+install.packages("/home/user/rdeephaven", repos=NULL, type="source")
+
+# load deephaven library
+library(rdeephaven)
+
+# connect to server using Client object
+# I hate this api more than Rcpp hates default arguments
+client <- new(Client, target="192.168.1.241:10000", session_type="python",
+                      auth_type="default", key="", value="")
+
+
+client$check_for_table("static_table2")
+
+# open DH table and convert to R dataframe
+static_table <- client$open_table("static_table1")
+data_frame1 <- dh_to_data_frame(static_table)
+data_frame1
+
+# modify dataframe with regular R stuff
+data_frame1$Name_Float_Col <- c(3.14, 2.71, 0.00)
+data_frame1
+
+# push new dataframe back to DH server and retrieve as DH table
+new_static_table <- df_to_dh_table(client, data_frame1, "static_table2")
```

Comment:
`client$push()` is hard at the moment because we need a pointer from the C++ side and a function call from the R side to make it work. I can't put the R function call directly into the C++ code to wrap everything in `$push()`, and I can't yet get into the S4 to put it there by hand. If you notice, there's not a single S4 class to be found anywhere in this source code. That is still black magic to me.